pkg/didcomm/common/service/errors.go

```diff
@@ -11,6 +11,8 @@ const (
 	ErrChannelRegistered = serviceError("channel is already registered for the action event")
 	ErrNilChannel        = serviceError("cannot pass nil channel")
 	ErrInvalidChannel    = serviceError("invalid channel passed to unregister the action event")
+	ErrThreadIDNotFound  = serviceError("threadID not found")
+	ErrInvalidMessage    = serviceError("invalid message")
```

Comment:
We shouldn’t expose errors types that aren’t expected to be checked and handled by consumer code. Are these errors that will be handled?

Reply:
why we shouldn't expose errors? For now, we do not check it, but I do not see any problems with that. These errors are safe. (I made them as constants, nobody can change it)

Reply:
Exposed errors become part of the API surface. Only errors that we expect developers to handle may be exposed. Otherwise do not expose them.